warmUp12.js

```diff
@@ -4,3 +4,9 @@
 
 // 2-write a function that takes array of strings and returns an array of the strings that have the same length
 // example :  ['hi','hello','welcome','hy'] == > ["hi","hy"]
+function sameLength(arr){
+	return filter (arr, function(elm, i){
```

Comment:
it gives me an error saying "Uncaught ReferenceError: filter is not defined", fo you know why ?

Reply:
we didn't reach filter yet, can you tell where did you learn about it?